.github/workflows/publish.nuget.yml

```diff
@@ -1,86 +1,78 @@
 name: Publish nuget package
```

Comment:
Please take a look into the workflow for nuget publishing that is being used in dotnet-utils repository [here](https://github.com/trakx/dotnet-utils/blob/dev/.github/workflows/nuget.yml).

Maybe you will be able to reuse some actions, instead of writing all by your own.

P.S.: Is you use trakx/github-actions, ensure you are using v9.1.5, that is the last version for .NET 6.

Reply:
I had nuget.yml in the file for a long time: https://github.com/trakx/kaiko-api-client/commit/42ea869d540bdb706f5d04b814e8f41866289c85#diff-bd6633be65e4d24ae2dd48955f22acf723749bda3921fcd1877ea0b8192ae0b6

I then deleted it because I could only run 'publish.nuget.yml' with a manual action, so having 2 files for the same thing seemed confusing to me. I'll add nuget.yml back.